chalicelib/checks/wrangler_checks.py

```diff
@@ -827,6 +828,8 @@ def finalize_user_pending_labs(connection, **kwargs):
     action_logs = {'patch_failure': [], 'patch_success': []}
     for user in check_res.get('full_output', []):
         patch_data = {'lab': user['pending_lab']}
+        if 'lab_PI_viewing_groups' in user:
```

Comment:
Since `user` will have `lab_PI_viewing_groups` value of None if the lab doesn't have a PI, I think this if-statement should probably be `if user.get('lab_PI_viewing_groups'):`

Reply:
^^ Otherwise, patching with None value for viewing groups will give validation error

Reply:
You are correct sir!  Forgot that fields in pending user got a default value of None.  Fixed.